### PR TITLE
Issue/#150 complete site settings functionality

### PR DESCRIPTION
**Pull request by korzol**
Hello @ceckoslab 
Here is latest changes. 
There is $settingsAliases  array in SiteSettingsController. According to best practices it would be better to create a objects to manipulate the data in that array. However at this moment it is this small and it is not known if it will be ever touched again.  So probably it is good to go with array..

### CODE REVIEW

**Comment by ceckoslab** [comment]
Hello @korzol 

It looks good.

I merged another task and now there are some Merge Conflicts. Could you fix them and then I can merge this one?

File: templates/admin/site_settings/index.html.twig [hunk 1]
```diff
@@ -38,17 +45,21 @@
...
-                alert("Something went wrong");
-            });
+    $().ready(function() {
+        $("#settings-form").validate({
+            errorLabelContainer: $("#form1 div.error"),
+            submitHandler: function(form) {
+                var formData = $('#settings-form').serialize();
+                $.post('{{ path('admin_site_settings_save') }}', formData)
+                    .done(function(response){
+                        console.log(response);
```
Comment (ceckoslab) [inline templates/admin/site_settings/index.html.twig:56]: Could we remove the console.log ?

File: templates/admin/site_settings/index.html.twig [hunk 2]
```diff
@@ -15,16 +15,23 @@
...
                     readonly="readonly"
-                    value="{{ aliases[item.name] }}"
+                    value="{{ aliases[item.name]['alias'] }}"
                     >
 
             </td>
             <td>
                 <input
-                    type="text"
+{#                    type="text"#}
```
Comment (ceckoslab) [inline templates/admin/site_settings/index.html.twig:24]: If we do not need this line just delete it.

File: templates/admin/site_settings/index.html.twig (hunk 1 above)
Comment (korzol) [reply templates/admin/site_settings/index.html.twig]: Done

File: templates/admin/site_settings/index.html.twig (hunk 2 above)
Comment (korzol) [reply templates/admin/site_settings/index.html.twig]: Done

**Pull request merged**